src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -383,9 +389,24 @@ def _sync_public_model_to_hub(self, model: JumpStartModelInfo, thread_num: int):
             f"{TASK_TAG_PREFIX}:TODO: pull from specs",
         ]
 
-        hub_content_document = HubModelDocument(
+        dest_file_dict: Dict[str, FileInfo] = {}
```

Comment:
q: May be not able to see it, but is this referenced somewhere?

Reply:
oops good catch ty